pkg/operator/configobservation/auth/auth_serviceaccountissuer.go

```diff
@@ -123,6 +127,34 @@ func observedConfig(
 	}, errs
 }
 
+// issuersChanged compares the command line flags used for KAS and the operator status service account issuers.
+// if these two sets are different, we have a change and we need to update the KAS.
+func issuersChanged(kasIssuers []string, activeOperatorIssuer string, trustedOperatorIssuers []string) bool {
```

Comment:
nit, this can be simplified for understandability as `issuersChanged(old []string, current ...string)`

and the caller-side: 
```
issuersChanged(
			existingIssuers,
			getActiveServiceAccountIssuer(operator.Status.ServiceAccountIssuers),
			getTrustedServiceAccountIssuers(operator.Status.ServiceAccountIssuers)...,
		)
```